### PR TITLE
Fit: added implementation of is object empty validator

### PR DESCRIPTION
**Pull request by ntwari-egide**

### CODE REVIEW

**Review by makuzaverite — CHANGES_REQUESTED** [review]
Add unit tests for this

File: src/areObjectsFromSameType.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  const keys2 = Object.entries(object2)
+
+  if (keys1.length !== keys2.length) {
+    return false
+  }
+
+  for (const key in keys1) {
+    if (object1[key] !== object2[key]) {
+      return false
```
Comment (makuzaverite) [inline src/areObjectsFromSameType.ts:20]: I can't find any where you are handling errors in this function

**Review by makuzaverite — CHANGES_REQUESTED** [review]
Add unit tests plz we use jest currently

**Comment by veritem** [comment]
@ntwari-egide  we might close this

**Pull request closed without merging**